SA/project_documentation/content/risk_management.typ

```diff
@@ -0,0 +1,39 @@
+#import "@preview/tablex:0.0.5": tablex, cellx
+
+= Risk Management
+The following section describes the risks we identified for our project and actions to prevent or correct them.
+Risks are color-coded according to @risk_matrix.
+
+== Initial Risk Assessment
+In @initial_risk_assessment all risks identified at the start of the project are documented.
+#figure(
+    tablex(
+        columns: 4,
+        align: center + horizon,
+
+        [*ID*], [*Risk*], [*Preventive actions*], [*Corrective actions*],
+
+        cellx(fill: green)[001], [*Difficulties with Typst*], [-], [Switch to LaTeX in case typst lacks neccessary features],
```

Comment:
We could add labels here, like

```suggestion
        cellx(fill: green)[001] <risk_001>, [*Difficulties with Typst*], [-], [Switch to LaTeX in case typst lacks neccessary features],
```

so that we can reference it later. We can always add it when we need it though, so you may merge this PR without it

Reply:
I tried to do this, but that doesn't seem to work like that. I can't put it after `]` because of a syntax error and if a put if before and try to reference it, I get `error: cannot reference text`.